packages/binding-opcua/src/opcua-protocol-client.ts

```diff
@@ -600,6 +613,63 @@ export class OPCUAProtocolClient implements ProtocolClient {
             const variantInJson = opcuaJsonEncodeVariant(dataValue.value, false);
             const content = contentSerDes.valueToContent(variantInJson, schemaDataValue, contentType);
             return content;
+        } else if (contentType === "application/octet-stream") {
```

Comment:
I think the general problem with the current code is that it right away decodes the data on the wire. It should first get the data `raw` and afterwards the ContentSerdes decodes the data.